pennylane/transforms/qcut.py

```diff
@@ -364,6 +365,158 @@ def graph_to_tape(graph: MultiDiGraph) -> QuantumTape:
     return tape
 
 
+def _get_measurements(
+    group: Sequence[Operator], measurements: Sequence[MeasurementProcess]
+) -> List[MeasurementProcess]:
+    """Pairs each observable in ``group`` with the fixed circuit ``measurements``.
+
+    Only a single fixed measurement of an expectation value is currently supported.
```

Comment:
What is a "fixed" circuit measurement, fixed as opposed to ?

Reply:
Fixed as in the measurements of the original circuit, rather than the product of Pauli terms